crates/contract_history/src/lib.rs

```diff
@@ -1,15 +1,11 @@
 pub const fn current_mainnet() -> &'static [u8] {
-    version_2_2_0()
+    version_3_0_2()
 }
 
 pub const fn current_testnet() -> &'static [u8] {
     version_3_0_2()
 }
 
-pub const fn version_2_2_0() -> &'static [u8; 566653] {
-    include_bytes!("../archive/signer-2_2_0.wasm")
-}
-
```

Comment:
Interesting. I pictured we'd retain the old contracts for historic reference, but I guess YAGNI applies here.